frontend/src/components/samples/SampleTableColumns.tsx

```diff
@@ -103,6 +105,55 @@ export const SAMPLE_COLUMN_DEFINITIONS: { [key in SampleColumnID]: SampleColumn
 		},
 	},
 
+	[SampleColumnID.PARENT_CONTAINER]: {
+		columnID: SampleColumnID.PARENT_CONTAINER,
+		title: 'Parent Container',
+		dataIndex: ['sample', 'container'],
+		render: (_, { sample }) => {
+			return (
+				sample &&
+				sample.container && (
+					<WithContainerRenderComponent
+						objectID={sample.container}
+						placeholder={<span>loading...</span>}
+						render={(container) =>
+							container.location ?
+								<WithContainerRenderComponent
+									objectID={container.location}
+									placeholder={<span>loading...</span>}
+									render={(container) => <Link to={`/containers/${container.id}`}>{container.barcode}</Link>}
+								/>
+								: <></>}
+					/>
+				)
+			)
+		},
+	},
+	
+	[SampleColumnID.PARENT_COORDINATES]: {
+		columnID: SampleColumnID.PARENT_COORDINATES,
+		title: 'Parent Coordinates',
+		dataIndex: ['sample', 'coordinate'],
+		render: (_, { sample }) => {
+			return (
+				sample &&
+				sample.container && (
+					<WithContainerRenderComponent
```

Comment:
If there is no parent container and thus no coordinate then we end up displaying "loading..." forever in the coordinate column.
<img width="1557" alt="Screen Shot 2023-09-12 at 4 40 24 PM" src="https://github.com/c3g/freezeman/assets/104078019/2a05b393-d819-455a-befb-529235120ff6">